### ISSUE TITLE
Error running ./paprica-run.sh

### ISSUE THREAD

**Issue by chongchunwie** [bowmanjeffs/paprica#71]
Dear Jeff,

I have issue running ./paprica-run.sh on the "test.fasta", the following error msg keep popping out. I dont have the issue previously, I am not sure what went wrong. For your info, I have tried using different PC and the issue persisted. 

INFO Output file: ./epa_result.jplace
Uncaught exception: syntax error parsing between line 1 character 0 and line 1 character 0 of /home/paprica/paprica/test.bacteria.combined_16S.bacteria.tax.clean.unique.align.jplace
Fatal error: exception Ppatteries.Sparse.Parse_error("syntax error parsing", _, _, _)
Uncaught exception: syntax error parsing between line 1 character 0 and line 1 character 0 of /home/paprica/paprica/test.bacteria.combined_16S.bacteria.tax.clean.unique.align.jplace
Fatal error: exception Ppatteries.Sparse.Parse_error("syntax error parsing", _, _, _)
Uncaught exception: syntax error parsing between line 1 character 0 and line 1 character 0 of /home/paprica/paprica/test.bacteria.combined_16S.bacteria.tax.clean.unique.align.jplace
Fatal error: exception Ppatteries.Sparse.Parse_error("syntax error parsing", _, _, _)
Traceback (most recent call last):
  File "/home/paprica/paprica/paprica-place_it.py", line 580, in <module>
    guppy_csv = pd.read_csv(cwd + query + '.' + ref + '.clean.unique.align.csv', header = 0, index_col = 'name')
  File "/home/paprica/.local/lib/python3.6/site-packages/pandas/io/parsers.py", line 702, in parser_f
    return _read(filepath_or_buffer, kwds)
  File "/home/paprica/.local/lib/python3.6/site-packages/pandas/io/parsers.py", line 429, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/paprica/.local/lib/python3.6/site-packages/pandas/io/parsers.py", line 895, in __init__
    self._make_engine(self.engine)
  File "/home/paprica/.local/lib/python3.6/site-packages/pandas/io/parsers.py", line 1122, in _make_engine
    self._engine = CParserWrapper(self.f, **self.options)
  File "/home/paprica/.local/lib/python3.6/site-packages/pandas/io/parsers.py", line 1853, in __init__
    self._reader = parsers.TextReader(src, **kwds)
  File "pandas/_libs/parsers.pyx", line 387, in pandas._libs.parsers.TextReader.__cinit__
  File "pandas/_libs/parsers.pyx", line 705, in pandas._libs.parsers.TextReader._setup_parser_source
FileNotFoundError: [Errno 2] File b'/home/paprica/paprica/test.bacteria.combined_16S.bacteria.tax.clean.unique.align.csv' does not exist: b'/home/paprica/paprica/test.bacteria.combined_16S.bacteria.tax.clean.unique.align.csv'
papric

**Comment by elijthomas** [comment]
I too encountered this issue today @bowmanjeffs running on linux python 3.7.7

**Comment by elijthomas** [comment]
`INFO     ______ ____   ___           _   __ ______
        / ____// __ \ /   |         / | / // ____/
       / __/  / /_/ // /| | ______ /  |/ // / __  
      / /___ / ____// ___ |/_____// /|  // /_/ /  
     /_____//_/    /_/  |_|      /_/ |_/ \____/ (v0.3.6)
mv: cannot stat 'epa_result.jplace': No such file or directory
Uncaught exception: Sys_error("/home/ubuntu/paprica/test.bacteria.combined_16S.bacteria.tax.clean.unique.align.jplace: No such file or directory")
Fatal error: exception Sys_error("/home/ubuntu/paprica/test.bacteria.combined_16S.bacteria.tax.clean.unique.align.jplace: No such file or directory")
Uncaught exception: Sys_error("/home/ubuntu/paprica/test.bacteria.combined_16S.bacteria.tax.clean.unique.align.jplace: No such file or directory")
Fatal error: exception Sys_error("/home/ubuntu/paprica/test.bacteria.combined_16S.bacteria.tax.clean.unique.align.jplace: No such file or directory")
Uncaught exception: Sys_error("/home/ubuntu/paprica/test.bacteria.combined_16S.bacteria.tax.clean.unique.align.jplace: No such file or directory")
Fatal error: exception Sys_error("/home/ubuntu/paprica/test.bacteria.combined_16S.bacteria.tax.clean.unique.align.jplace: No such file or directory")
Traceback (most recent call last):
  File "/home/ubuntu/paprica/paprica-place_it.py", line 580, in <module>
    guppy_csv = pd.read_csv(cwd + query + '.' + ref + '.clean.unique.align.csv', header = 0, index_col = 'name')
  File "/home/linuxbrew/.linuxbrew/Cellar/python/3.7.7_1/lib/python3.7/site-packages/pandas/io/parsers.py", line 676, in parser_f
    return _read(filepath_or_buffer, kwds)
  File "/home/linuxbrew/.linuxbrew/Cellar/python/3.7.7_1/lib/python3.7/site-packages/pandas/io/parsers.py", line 448, in _read
    parser = TextFileReader(fp_or_buf, **kwds)
  File "/home/linuxbrew/.linuxbrew/Cellar/python/3.7.7_1/lib/python3.7/site-packages/pandas/io/parsers.py", line 880, in __init__
    self._make_engine(self.engine)
  File "/home/linuxbrew/.linuxbrew/Cellar/python/3.7.7_1/lib/python3.7/site-packages/pandas/io/parsers.py", line 1114, in _make_engine
    self._engine = CParserWrapper(self.f, **self.options)
  File "/home/linuxbrew/.linuxbrew/Cellar/python/3.7.7_1/lib/python3.7/site-packages/pandas/io/parsers.py", line 1891, in __init__
    self._reader = parsers.TextReader(src, **kwds)
  File "pandas/_libs/parsers.pyx", line 374, in pandas._libs.parsers.TextReader.__cinit__
  File "pandas/_libs/parsers.pyx", line 674, in pandas._libs.parsers.TextReader._setup_parser_source
FileNotFoundError: [Errno 2] File /home/ubuntu/paprica/test.bacteria.combined_16S.bacteria.tax.clean.unique.align.csv does not exist: '/home/ubuntu/paprica/test.bacteria.combined_16S.bacteria.tax.clean.unique.align.csv`

**Comment by elijthomas** [comment]
This seems to only occur on a single core box for me ?

**Comment by bowmanjeffs** [comment]
Hmmm... just back from a long research cruise so I didn't see the March 1 bug report.  The two errors looks slightly different.  Let me see if I can reproduce...

**Comment by bowmanjeffs** [comment]
I can't reproduce the error.  One thought is that if you're running on a single-core box (which would be pretty uncommon) you might have run out of memory.  It looks like epa-ng didn't run successfully.  So check memory utilization during the run, and always run paprica on a system with sufficient memory (I suggest 8 Gb - troubleshooting this I noticed that our VM testbox with 6 Gb no longer has sufficient memory for the EDPL step.  This will happen as the tree grows).  Closing the thread but re-open if problem persists.
